spec/inputs/file_spec.rb

```diff
@@ -49,22 +47,25 @@
 
         if queue.size >= 2
           events = 2.times.collect { queue.pop }
-          insist { events[0]["message"] } == "hello"
-          insist { events[1]["message"] } == "world"
           break
         end
 
         sleep(0.1)
         retries += 1
       end
+
+      events
     end
+
+    insist { events[0]["message"] } == "hello"
```

Comment:
can we use this PR to also get rid of insist in here? it would be a big clean up :-) but if you don't want to do it in here I do understand.